server/src/main/scala/net/wiringbits/repositories/daos/UsersDAO.scala

```diff
@@ -64,6 +64,14 @@ object UsersDAO {
     """.execute()
   }
 
+  def resetPassword(userId: UUID, password: String)(implicit conn: Connection): Unit = {
+    val _ = SQL"""
+      UPDATE users
+      SET password = $password
```

Comment:
I think we have an `updated_at` column that must be updated too

Reply:
It seems like we don't have it. Should I add it?